documentation/modules/managing/con-consumer-config-properties.adoc

```diff
@@ -250,6 +250,19 @@ A shorter interval between consecutive heartbeats allows for quicker detection o
 The heartbeat interval must be lower, usually by a third, than the session timeout. 
 Decreasing the heartbeat interval reduces the chance of accidental rebalancing, but more frequent heartbeats increases the overhead on broker resources.
 
+[source,env]
+----
+# ...
+max.poll.records=500 # <1>
+session.timeout.ms=45000 # <2>
+heartbeat.interval.ms=3000 # <3>
+# ...
+----
+<1> Sets the number of processed records returned from the consumer.
```

Comment:
This is the maximum number of records returned when calling `poll()`. These are not processed but handed to the consumer to process them.